recon_test_pack/run_test_simulate_and_recon.sh

```diff
@@ -133,26 +135,30 @@ for recon in FBP2D FBP3DRP SRT2D SRT2DSPECT OSMAPOSL OSSPS ; do
         is_analytic=1
       elif expr "$recon" : SRT > /dev/null; then
         is_analytic=1
+      elif expr "$recon" : GRD > /dev/null; then
+        is_analytic=1
+      elif expr "$recon" : DDSR > /dev/null; then
+        is_analytic=1		
       fi
       if [ $is_analytic = 1 ]; then
-          if expr "$dataSuffix" : '.*TOF.*' > /dev/null; then
-            echo "Skipping TOF as not yet supported for FBP and SRT"
-            break
-          fi
-	  if expr "$recon" : SRT2DSPECT > /dev/null; then
-	    suffix=$SPECT_suffix
-	    export suffix
-          else   
-            suffix=$zero_view_suffix
-            export suffix
-            echo "Running precorrection"
-	    correct_projdata correct_projdata_simulation.par > my_correct_projdata_simulation.log 2>&1
-	    if [ $? -ne 0 ]; then
-              echo "Error running precorrection. CHECK my_correct_projdata_simulation.log"
-	      error_log_files="${error_log_files} my_correct_projdata_simulation.log"
-	      break
-	    fi
-          fi
+        if expr "$dataSuffix" : '.*TOF.*' > /dev/null; then
+          echo "Skipping TOF as not yet supported for FBP, SRT, GRD and DDSR."
+          break
+        fi
+		  if expr "$recon" : '\(SRT2DSPECT\|DDSR2D\)' > /dev/null; then
```

Comment:
Given the run-time error on MacOS, this is the most likely place for failure. Presumably, `expr` doesn't do (the same) pattern matching on MacOS. https://ss64.com/mac/expr.html refers to a page on `re_format` which I cannot find unfortunately. However, the regular expression looks overly complicated to me. On Linux, the following works as well
```suggestion
		  if expr "$recon" : 'SRT2DSPECT\|DDSR2D' > /dev/null; then
```